components/Dashboard/Post/Post.tsx

```diff
@@ -539,9 +539,25 @@ const Post: React.FC<IPostProps> = ({ post, currentUser, refetch }: IPostProps)
           cursor: pointer;
           border-radius: 3px;
         }
+
         .thread-highlight:hover {
           background-color: ${theme.colors.highlightColorHover};
         }
+
+        .post-body p {
+          margin: 5px 0;
+        }
+
+        .post-body table {
+          width: 100%;
+        }
+
+        .post-body td {
+          background-color: rgb(255, 255, 255);
+          border: 1px solid rgb(193, 199, 208);
```

Comment:
It would be great if we could add these to our color palette / theme (see line 544 just above in this file). Also, this is a small detail but we usually use HEX codes so I wonder if we should try to use those here just for consistency

Reply:
For sure I'll make this refactoring!